README.md

```diff
@@ -16,6 +16,14 @@ I recommend creating a separate package inside your project to serve as the cont
 
 For an example of how to use the resulting package, check out `example/example.go`. You can also see the API it generates at [godoc.org](https://godoc.org/github.com/bouk/staticfiles/files).
 
+## Installation
+
+Install with
+
+```
+go get github.com/bouk/staticfiles
```

Comment:
But having said that I think there is no harm in adding `-u`. It depends on the author of the project. 

But `-u` flag is generally associated with updating your library and its dependencies. 
From https://golang.org/cmd/go/
```
The -u flag instructs get to use the network to update the named packages and their dependencies. By default, get uses the network to check out missing packages but does not use it to look for updates to existing packages.
```

And in a project's readme, I guess without `-u` flag is just fine because here we are just giving a step for installation.